src/main/kotlin/org/elaastic/questions/player/PlayerController.kt

```diff
@@ -723,4 +722,60 @@ class PlayerController(
         val confidenceDegree: ConfidenceDegree?,
         val explanation: String?
     )
+
+    /**
+     * Get the result view for the given learner
+     *
+     * @param authentication the current authentication
+     * @param model the model
+     * @param userId the id of the learner to get the result view
+     * @return the result view for the given learner
+     */
+    @GetMapping("/{sequenceId}/result/{userId}")
+    fun result(
+        authentication: Authentication,
+        model: Model,
+        @PathVariable sequenceId: Long,
+        @PathVariable userId: Long
+    ): String {
+        val user: User = authentication.principal as User
+
+        val sequence = sequenceService.get(sequenceId, true)
+        val learner = userService.findById(userId)
+
+        // TODO change requireAccess to requireAccessThrowDenied when fix/249-cant-see-chatgpt-feedback-with-likert will be merged
```

Comment:
Is-it possible to fix this TODO now ?

Reply:
Fix in e1d442f5532d0f6068acc657326bc661b774e12e